### PR TITLE
[Private DC] GKE deployment config and util functions update to make it run

### PR DESCRIPTION
**Pull request by shifucun**
- There is no provenance from TMCF+CSV, may need to enforce a manifest for each import.
- Color change is not reflected in this PR. Will do a separate one for color and homepage change.

### CODE REVIEW

File: server/__init__.py [hunk 1]
```diff
@@ -122,7 +122,7 @@ def create_app():
     app.config['BABEL_TRANSLATION_DIRECTORIES'] = 'i18n'
 
     if not cfg.TEST:
-        timeout = 120  # seconds
+        timeout = 60  # seconds
```
Comment (beets) [inline server/__init__.py:125]: i don't remember why we bumped up the timeout - were there long API calls that we rely on? even so, best to do this in a separate PR

File: server/webdriver_tests/base_test.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ def setUp(self):
         chrome_options.add_argument('--disable-dev-shm-usage')
 
         # Maximum time, in seconds, before throwing a TimeoutException.
-        self.TIMEOUT_SEC = 60
+        self.TIMEOUT_SEC = 120
```
Comment (beets) [inline server/webdriver_tests/base_test.py:52]: ditto, please move this to a new PR

File: server/templates/base.html [hunk 3]
```diff
@@ -29,7 +30,7 @@
 -#}
 <!DOCTYPE html>
 
-<html lang={{ locale }}>
+<html lang={{ locale }} class={{ DEPLOYMENT }}>
```
Comment (beets) [inline server/templates/base.html:33]: i'd prefer it if we only set an overriding class in the private case, so public remains the default (besides some performance concerns, also to avoid us relying on the public class)

**Review by beets — APPROVED** [review]
very cool bo!

File: server/webdriver_tests/base_test.py (hunk 2 above)
Comment (shifucun) [reply server/webdriver_tests/base_test.py]: Done

File: server/templates/base.html (hunk 3 above)
Comment (shifucun) [reply server/templates/base.html]: Done

File: server/__init__.py (hunk 1 above)
Comment (shifucun) [reply server/__init__.py]: Done
